### ISSUE TITLE
Cannot download "https://github.com/sass/node-sass/releases/download/v3.13.1/linux-x64-57_binding.node":

### ISSUE THREAD

**Issue by gigo6000** [sass/node-sass#3417]
Our build is failing with because of this error:

```
Downloading binary from https://github.com/sass/node-sass/releases/download/v3.13.1/linux-x64-57_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v3.13.1/linux-x64-57_binding.node":

HTTP error 404 Not Found
```
We have a legacy app on `prod` running this old version and we're in the process of migrating to the latest version of `node` and `Angular` but in the meantime we need to keep this one running. ~~Was this release of node-sass `v3.13.1` deleted from the repo ?~~ It looks like the release is there but not the asset. Any help is appreciate it. Thank you!

- NPM version (`npm -v`): 2.15.11
- Node version (`node -v`): v4.9.1
- Node Process (`node -p process.versions`): { http_parser: '2.8.0',
  node: '4.9.1',
  v8: '4.5.103.53',
  uv: '1.9.1',
  zlib: '1.2.11',
  ares: '1.10.1-DEV',
  icu: '56.1',
  modules: '46',
  openssl: '1.0.2o' }
- Node Platform (`node -p process.platform`): linux
- Node architecture (`node -p process.arch`): x64
- node-sass version (`node -p "require('node-sass').info"`):node-sass	3.13.1	(Wrapper)	[JavaScript]
libsass  	3.3.6	(Sass Compiler)	[C/C++]
- npm node-sass versions (`npm ls node-sass`): └─┬ gulp-sass@2.3.2
  └── node-sass@3.13.1

**Comment by saper** [comment]
You must using node 8.x somewhere in the process, that's what the "57" number stands for.

**Comment by gigo6000** [comment]
Yes, we actually use node `v8.11.2` not sure where the other version came from.  But anyway we fixed it upgrading `gulp-sass` to `4.0.0` in case anyone else has this issue.
